libs/perun/utils/src/lib/export-table-dialog/export-table-dialog.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, Inject, ViewEncapsulation } from '@angular/core';
+import { FormControl, Validators } from '@angular/forms';
+import { MAT_DIALOG_DATA } from '@angular/material/dialog';
+
+interface Format {
+  value: string;
+  viewValue: string;
+}
+
+interface InputData {
+  allowExportAll: boolean;
+}
+
+@Component({
+  selector: 'perun-web-apps-export-table-dialog',
+  templateUrl: './export-table-dialog.component.html',
+  styleUrls: ['./export-table-dialog.component.scss'],
+  encapsulation: ViewEncapsulation.None,
+})
+export class ExportTableDialogComponent {
+  formats: Format[] = [{ value: 'csv', viewValue: 'CSV' }];
+  selectedFormat = new FormControl<string | null>(null, Validators.required);
+  selectedExportType = new FormControl<string | null>(null, Validators.required);
```

Comment:
I think it would be nice to have default values preselected. Specially since there is just one option in some cases.
```suggestion
  selectedFormat = new FormControl<string>('csv', Validators.required);
  selectedExportType = new FormControl<string>('current', Validators.required);
```